### PR TITLE
Add live update checking

### PR DESCRIPTION
**Pull request by rtm516**
TODO:
- [x] Comment code
- [ ] ~Possibly add bedrock release detection~

### CODE REVIEW

File: bot_modules/live_updates/jira-je.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  return response.data
+}
+
+/**
+ * Take a version object and fomat it into a nice message for discord
+ *
+ * @param {Object} version The version data to insert into the message
+ */
+function jiraAsString (version) {
+  return `A new java version (${version.name}) has been added to the minecraft issue tracker!`
```
Comment (DoctorMacc) [inline bot_modules/live_updates/jira-je.js:63]: Capitalize Java here.

File: bot_modules/live_updates/jira-be.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  return response.data
+}
+
+/**
+ * Take a version object and fomat it into a nice message for discord
+ *
+ * @param {Object} version The version data to insert into the message
+ */
+function jiraAsString (version) {
+  return `A new bedrock version (${version.name}) has been added to the minecraft issue tracker!`
```
Comment (DoctorMacc) [inline bot_modules/live_updates/jira-be.js:63]: Capitalize Bedrock here.

File: .env.example [hunk 3]
```diff
@@ -1,3 +1,4 @@
 BOT_TOKEN=your bot token
 CONFIG_EDITING_USERS=["502575111346323456", "229380062909038594", "119922034812780547", "267678060914933770", "170429173003714560", "245958554899578880"]
-CONFIG_EDITING_GROUPS=["737810664034271302"]
\ No newline at end of file
+CONFIG_EDITING_GROUPS=["737810664034271302"]
+DISCORD_UPDATE_CHANNELS=["349264069007638530"]
```
Comment (Heath123) [inline .env.example:4]: Shouldn't this be in the config as it's not sensitive or needed to edit the config itself if it goes wrong?

File: bot_modules/live_updates/index.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+const jiraJE = require('./jira-je')
+const jiraBE = require('./jira-be')
+const minecraft = require('./minecraft')
+
+exports.init = async (client) => {
```
Comment (Heath123) [inline bot_modules/live_updates/index.js:5]: Switching to arrow functions should probably be a seperate PR

**Pull request merged**